app/scripts/lib/account-tracker.test.js

```diff
@@ -451,114 +450,16 @@ describe('Account Tracker', () => {
     });
   });
 
-  describe('_updateAccount', () => {
```

Comment:
Just want to confirm: Have we checked tests for methods that call `_updateAccount` to see whether they include these?

Reply:
yes, this got folded into the scenarios in updateAccount